source/access/mfa_guide.rst

```diff
@@ -3,6 +3,27 @@
 MFA Guide
 =========
 
+General information
+-------------------
+
+Mutli-Factor Authentication (MFA) is mandatory when connecting to the VSC network.
```

Comment:
Mutli -> Multi

Also note that this is not enforced everywhere, so perhaps better to use the "|KUL|" tag for now?